cmd/controller/main.go

```diff
@@ -38,15 +38,16 @@ const (
 )
 
 var (
-	image              = flag.String("el-image", elresources.DefaultImage, "The container image for the EventListener Pod.")
-	port               = flag.Int("el-port", elresources.DefaultPort, "The container port for the EventListener to listen on.")
-	setSecurityContext = flag.Bool("el-security-context", elresources.DefaultSetSecurityContext, "Add a security context to the event listener deployment.")
-	readTimeOut        = flag.Int64("el-readtimeout", elresources.DefaultReadTimeout, "The read timeout for EventListener Server.")
-	writeTimeOut       = flag.Int64("el-writetimeout", elresources.DefaultWriteTimeout, "The write timeout for EventListener Server.")
-	idleTimeOut        = flag.Int64("el-idletimeout", elresources.DefaultIdleTimeout, "The idle timeout for EventListener Server.")
-	timeOutHandler     = flag.Int64("el-timeouthandler", elresources.DefaultTimeOutHandler, "The timeout for Timeout Handler of EventListener Server.")
-	periodSeconds      = flag.Int("period-seconds", elresources.DefaultPeriodSeconds, "The Period Seconds for the EventListener Liveness and Readiness Probes.")
-	failureThreshold   = flag.Int("failure-threshold", elresources.DefaultFailureThreshold, "The Failure Threshold for the EventListener Liveness and Readiness Probes.")
+	image                 = flag.String("el-image", elresources.DefaultImage, "The container image for the EventListener Pod.")
+	port                  = flag.Int("el-port", elresources.DefaultPort, "The container port for the EventListener to listen on.")
+	setSecurityContext    = flag.Bool("el-security-context", elresources.DefaultSetSecurityContext, "Add a security context to the event listener deployment.")
+	setEventListenerEvent = flag.Bool("el-events", elresources.DefaultEventListenerEvent, "Enable events for event listener deployment.")
```

Comment:
What do you think of making this flag a string instead of a bool? The values for now would be "enable" and "disable". We could, in the future, add values to selectively send only some events and not others.

Reply:
Sure i can make it as string
naive q: 

> We could, in the future, add values to selectively send only some events and not others.

do you mean adding values to string variable ?
and what kind of values we would expect :thinking:

Reply:
enable, disable, only-success-events, all-events etc. things like that

Reply:
Ah i see :+1: 
Updated PR with string instead of bool
PTAL
Thank you